gene2phenotype_project/gene2phenotype_app/serializers/attrib.py

```diff
@@ -6,13 +6,46 @@ class AttribTypeSerializer(serializers.ModelSerializer):
     """
         Return the list of attribs for a specific type.
     """
-
+    # to do - the way we presently fetch mutation mechanism is wrong 
```

Comment:
Do you mean the `mutation_consequence`? This attrib is deprecated/deleted, we should use the new flag you added in this PR to flag this attrib and not return it.

Reply:
The molecular mechanism is not an attrib anymore, we fetch the mechanism in a separate endpoint.

Reply:
Yeah but I was making the note for myself to remember to update description fetching for mutation endpoint because we would need it for the terminology page also

Reply:
Good point. 
If you want you can add the updates to this PR. The molecular mechanism is not called `attrib` but it's part of the same concept.